src/game.zig

```diff
@@ -1110,6 +1119,7 @@ pub fn update(deltaTime: f64) void { // MARK: update()
 			// This calculates how long the player has to fall until we know they're not walking over a small gap.
 			// We add deltaTime because we subtract deltaTime at the bottom of update
 			Player.eyeCoyote = @sqrt(2 * Player.steppingHeight()[2] / gravity) + deltaTime;
+			Player.jumpCoyote = Player.jumpCoyoteTimeConstant;
```

Comment:
Setting it here should be enough. I don't think you need to reset the jumpCoyote anywhere else.